wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/WebSocketRequestHandler.java

```diff
@@ -97,6 +128,36 @@ public void push(byte[] message, int offset, int length)
 		}
 	}
 
+	@Override
+	public Future<Void> pushAsync(byte[] message, int offset, int length)
```

Comment:
I would replace this method with `return pushAsync(message, offset, length, -1)`